cleverhans/utils_eager.py

```diff
@@ -0,0 +1,252 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from distutils.version import LooseVersion
+import math
+import numpy as np
+import os
+from six.moves import xrange
+import tensorflow as tf
+import time
+import warnings
+
+from .utils import batch_indices, _ArgsWrapper, create_logger
+
+_logger = create_logger("cleverhans.utils.eager")
+
+
+def model_loss(logits, labels, mean=True):
```

Comment:
What is specific to eager in this method? It looks very similar to the one in utils.tf

Reply:
True, will import it.